File: FromOutsideSources/AngularSecurityUsingDotNET5andJSONWebTokens/PTC/src/app/product/product.ts
export class Product {
  public constructor(init?: Partial<Product>) {
    Object.assign(this, init);
  }

  productId: number = 0;
  productName: string = "";
  introductionDate: Date = new Date();
  price: number = 0;
  url: string = "";
  categoryId: number = 0;
}

File: FromOutsideSources/AngularSecurityUsingDotNET5andJSONWebTokens/PTCApi/EntityClasses/Product.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PTCApi.EntityClasses
{
  [Table("Product", Schema = "dbo")]
  public partial class Product
  {
    public int ProductId { get; set; }

    public string ProductName { get; set; }

    public DateTime? IntroductionDate { get; set; }

    public decimal? Price { get; set; }

    public string Url { get; set; }

    public int? CategoryId { get; set; }
  }
}
